submariner-operator/issue-359-subctl-info-status.md

```diff
@@ -0,0 +1,93 @@
+## Add more capabilities to check status from subctl
+
+https://github.com/submariner-io/submariner-operator/issues/359
+
+
+## Summary
+The current info command is very limited and does not provide any information on current deployment status, only the discovered network details.
+```bash 
+subctl info  
+  Discovered network details:  
+        Network plugin:  weave-net  
+        Service CIDRs: [100.92.0.0/16]  
+        Cluster CIDRs: [10.242.0.0/16]
+```
+
+## Motivation
+To supply more information on current deployment status
+
+## Proposal
+```bash
+subctl show networks  
+    Discovered network details:  
+         Network plugin:  weave-net  
+         Service CIDRs: [100.92.0.0/16]  
+         Cluster CIDRs: [10.242.0.0/16]  
+         GlobalCIDR:    [<>] 
+```
+
+
+
+``` bash 
+subctl show gateways  
+NODE       HA-STATUS       CONNECTIONS    SUMMARY  
+worker1    active          3/3            All connections OK  
+worker2    passive         0/0                                
+```  
+    
+```bash
+subctl show connections  
+GATEWAY  CLUSTER    REMOTE-IP    DRIVER     SUBNETS                         STATUS  
+worker1  cluster2   172.17.0.10  libreswan  100.93.0.0/16,  10.243.0.0/16   connected  
+worker1  cluster3   172.17.0.11  libreswan  100.94.0.0/16,  10.244.0.0/16   connected
+```
+  
+  
+```bash 
+subctl show endpoints  
```

Comment:
Let's show this also in tabular format. ^ 
```
CLUSTER     IP    PUBLIC-IP    CABLE-DRIVER
```
endpoints in plural would mean that we are listing all endpoints in the database, not just our local endpoint.

May be we also want to add `subctl show local-endpoint` in the current format you are currently explaining here?

Reply:
IMO, show endpoints can display both local and remote endpoints.

Reply:
A `TYPE` field with local/remote works better.

Reply:
I also think TYPE local/remote could work better, let me add the suggestion here and let's ask maayan and sridhar.

Reply:
Yeah, Type with local/remote looks more readable.